lametro/templates/lametro/event.html

```diff
@@ -55,9 +55,9 @@ <h4>Participants</h4>
                     </p>
                     {% endif %}
               
-                    {% if agenda_url or uploaded_agenda_url %}
+                    {% if agenda_url or uploaded_agenda_url or uploaded_agenda_pdf %}
 
-                    {% if uploaded_agenda_url and not agenda_url%}
+                    {% if uploaded_agenda_url or uploaded_agenda_pdf and not agenda_url%}
```

Comment:
I'm not sure about Django templates but Python evaluates `and` before `or`, so this line would be like `if uploaded_agenda_url or (uploaded_agenda_pdf and not agenda_url)`. 

If we mean to say `if there is an uploaded URL or PDF, and there is not an agenda URL`, I think we need to put the `or` clause in parentheses.

Reply:
This can be simplified to: `{% if not agenda_url%}`...since we check for `uploaded_agenda_url or uploaded_agenda_pdf` in previous template logic.